packages/babel-core/src/transformation/index.js

```diff
@@ -64,7 +64,7 @@ export function runSync(
   return {
     metadata: file.metadata,
     options: opts,
-    ast: opts.ast !== false ? file.ast : null,
```

Comment:
Most callers of Babel don't use this and it is a huge structure. Not including it in the output means when we land caching logic in core one day, we'll have less data to cache by default.

Reply:
o